### PR TITLE
[RFR] US-11 e2e tests

### PR DESCRIPTION
**Pull request by djhi**
[Trello card](https://trello.com/c/Lt2nMPKB/11-implementer-des-tests-e2e)

- [x] Bootstrapped e2e tests
- [x] Made sure we can inject config correctly

### CODE REVIEW

**Comment by Kmaschta** [comment]
Great work!

Don't forget the test-dist.json

File: src/index.js [hunk 1]
```diff
@@ -7,9 +7,8 @@ import github from 'octonode';
 import githubClientFactory from './git/clients/github';
 import loggerFactory from './lib/logger';
```
Comment (alexisjanvier) [inline src/index.js]: Did we even need this ?

File: src/index.js [hunk 2]
```diff
@@ -7,9 +7,8 @@ import github from 'octonode';
...
 import parserFactory from './parser';
-const logger = loggerFactory(config);
 
-const main = function* (event, context) {
+const main = function* (event, context, logger) {
```
Comment (alexisjanvier) [inline src/index.js]: Where does this `logger` ?

Comment (djhi) [inline src/index.js]: it logs debug info all over the place

File: src/index.js (hunk 1 above)
Comment (djhi) [inline src/index.js]: yes we do, for debugging purpose

**Pull request merged**